src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -132,7 +133,7 @@ public Hardware() {
             frontRightModule = FRONT_RIGHT_CONFIG.falcons();
             backLeftModule = BACK_LEFT_CONFIG.falcons();
             backRightModule = BACK_RIGHT_CONFIG.falcons();
-            pigeon = new Pigeon(GYRO_PORT);
+            gyro = new NavX(SPI.Port.kMXP);//new Pigeon(GYRO_PORT);
```

Comment:
This should also be either reverted or programmatically changed based on which robot this is running on.